src/pages/listings/[type]/[slug]/winner.tsx

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import type { GetServerSideProps } from 'next';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+
+import { type Bounty } from '@/features/listings';
+import { getURL } from '@/utils/validUrl';
+
+interface BountyDetailsProps {
+  bounty: Bounty | null;
+}
+
+function WinnerBounty({ bounty: initialBounty }: BountyDetailsProps) {
+  const [bounty] = useState<typeof initialBounty>(initialBounty);
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push(`${getURL()}listings/${bounty?.type}/${bounty?.slug}/`);
+  }, []);
+
+  return (
+    <Head>
+      <title>{`Superteam Earn Bounty | ${
+        initialBounty?.title || 'Apply'
+      } by ${initialBounty?.sponsor?.name}`}</title>
+      <meta
+        name="description"
+        content={`Bounty on Superteam Earn | ${
```

Comment:
had made this change yesterday, so you might have missed it
replace this line with ```content={`${getBountyTypeLabel(initialBounty?.type ?? 'Bounty')} on Superteam Earn | ......```